CONTRIBUTING.md

```diff
@@ -19,10 +19,28 @@ The core team will be monitoring for pull requests. When we get one, we'll run s
 1. Fork the repo and create your branch from `master`.
 2. If you've added code that should be tested, add tests!
 3. If you've changed APIs, update the documentation.
-4. Ensure tests pass on Travis.
-5. Make sure your code lints (`node linter.js <files touched>`).
-6. Squash your commits (`git rebase -i`).
-7. If you haven't already, complete the CLA.
+4. Add the copyright notice to the top of any new files you've added.
+5. Ensure tests pass on Travis.
+6. Make sure your code lints (`node linter.js <files touched>`).
```

Comment:
Not related to your PR, but where is this `linter.js` file? Isn't the correct way to do this will be to run `eslint`?